pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "domaintools_api"
+version = "1.0.1"
```

Comment:
As per discussion last Retro, I think this should be 1.1.0?

Reply:
Yes, I'll be pushing the update together with my next commit. thanks JD!